boost_mailing_list_tracker/tests/fixtures.py

```
@@ -0,0 +1,49 @@
+"""
+
+from datetime import datetime, timezone
+
+import pytest
+from model_bakery import baker
+
+from boost_mailing_list_tracker.models import MailingListName
+
+
+@pytest.fixture
+def mailing_list_profile(db, identity):
```

Ruff (0.15.2)

[warning] Unused function argument: `db`

(ARG001)